packages/meteor/url_server.js

```diff
@@ -4,7 +4,7 @@ if (process.env.ROOT_URL &&
   if (__meteor_runtime_config__.ROOT_URL) {
     var parsedUrl = Npm.require('url').parse(__meteor_runtime_config__.ROOT_URL);
     // Sometimes users try to pass, eg, ROOT_URL=mydomain.com.
-    if (!parsedUrl.host) {
+    if (['http:', 'https:'].indexOf(parsedUrl.protocol) === -1 || !parsedUrl.host) {
```

Comment:
Why not reverse this condition, like so?
```js
if (!parsedUrl.host || ['http:', 'https:'].indexOf(parsedUrl.protocol) === -1) {
```

Reply:
@benjamn I'd say both are of the same importance, so order doesn't matter. I put the protocol test first because it's the way we read URLs, but it can be reversed if you prefer.

By any chance, have you seen my question here: https://github.com/meteor/meteor/pull/9261#issuecomment-342100497 ?

Reply:
@BDav24 I believe Ben's comment was suggesting to reverse the condition as there is a small performance gain when doing the [falsy check first](https://jsperf.com/falsy-vs-indexof/1).

Reply:
Well you'll gain 0.15 microseconds (0.00015 milliseconds) each time you start an instance :)